### DISCUSSION TITLE
Discussion paper content collection

### DISCUSSION THREAD

**Discussion by christianheimann** [e-id-admin/open-source-community#25] [General]
Goal is to have a public discussion around the tech decision for our eID and trust infrastructure. Therefore the eID-Team is currently preparing a discussion paper with our current knowledge and possible scenarios. The outcomes from the current activities in the Technical Advisory Circle (TAC) will be used as inputs for this paper aswell.

In this thread below, you fill find the **draft of the document structure**; one sub thread per chapter -- so comments can directly go under the concerned thread.

Also, as an important element, one subthread contains valuable **references on resources** (papers, studies, standards...) the team should consider. Feel free to add your reference documents.

Index:
0 References/Sources
1 Purpose of the document
2 Background
3 Design Principles & Criteria
4 Scenarios
5 Public discussion of this paper and further process

_Please place general comments at the last sub-thread._

**Comment by christianheimann** [comment]
**0) References/Sources**

**Reply by christianheimann** [reply]
EU-Architectural Reference Framework (ARF):
https://digital-strategy.ec.europa.eu/en/library/european-digital-identity-wallet-architecture-and-reference-framework

**Reply by christianheimann** [reply]
A GitHub thread that raised privacy concerns with regard to the EU-ARF:
https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/issues/66

**Reply by christianheimann** [reply]
Credential Format Comparison:
https://openwallet-foundation.github.io/credential-format-comparison-sig

And its preceding work:
https://docs.google.com/spreadsheets/d/1Z4cYfjbbE-rABcfC-xab8miocKLomivYMUFibOh9BVo
https://hackmd.io/ikhimdHLRy247YgRhNIXmw?both

**Reply by svenstucki** [reply]
https://portal.etsi.org/webapp/WorkProgram/Report_WorkItem.asp?WKI_ID=67975

ETSI technical report comparing selective disclosure mechanisms including unlinkability.

**Reply by christianheimann** [reply]
OIX, mentionned in a TAC meeting, having a Whitepaper for "Interoperability across trust frameworks"
https://openidentityexchange.org/papers

**Reply by bjafs** [reply]
(Some recent) Papers related to Credentials and BBS+ Signatures published in the _Cryptology ePrint Archive_:
https://eprint.iacr.org/2023/275
https://eprint.iacr.org/2023/602
https://eprint.iacr.org/2023/1076

**Reply by christianheimann** [reply]
Nach dem Entscheid wäre eine solch schöne Umsetzungs-Dokumentation anzustreben:
https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/index.html

**Reply by christianheimann** [reply]
Slides and meeting minutes from all technical advisory circle meetings
https://github.com/e-id-admin/general/blob/main/meetings/20230921_TAC_Technical_Advisory_Circle_1.pdf
https://github.com/e-id-admin/general/blob/main/meetings/20230921_TAC_Meetingminutes.pdf
https://github.com/e-id-admin/general/blob/main/meetings/20231016_TAC_Technical_Advisory_Circle_2.pdf
https://github.com/e-id-admin/general/blob/main/meetings/20231016_TAC_Meetingminutes.pdf
https://github.com/e-id-admin/general/blob/main/meetings/20231109_TAC_Technical_Advisory_Circle_3.pdf
https://github.com/e-id-admin/general/blob/main/meetings/20231109_TAC_Meetingminutes.pdf

**Reply by christianheimann** [reply]
ToIP: Simplifying Protocol Selection
https://trustoverip.org/blog/2023/10/10/simplifying-the-selection-process-for-credential-issuance-and-presentation-protocols/

**Reply by christianheimann** [reply]
Validity Credentials
https://eprint.iacr.org/2022/1658.pdf

**Comment by christianheimann** [comment]
**1) Purpose of the document**
Over the last two years, the Swiss Government was able to bring together a community to develop a shared vision of the future Swiss e-ID and its underlying trust infrastructure. Soon it will be facing an important juncture to decide on the cornerstones “how” the infrastructure enabling this vision will be built. Including a broad community is a great opportunity to discuss potential scenarios and gather insights regarding the decision. This document aims to create an overview and bring up possibilities for a technological basis, while highlighting benefits and drawbacks as transparent as possible. It is meant to kick off diverse discussions and enable the confederation to gain statements and feedback from interested parties in the public. Even though the content of this document is technical, additionally political and societal aspects will be drivers for the tech decision.

**Comment by christianheimann** [comment]
**2) Background**
Outline of the vision for a Swiss e-ID and  trust infrastructure for verifiable credentials.
Law making, tech neutrality/agnosticism; architecture and roles are the guiding principles.


EU eIDAS 2.0 and Architecture Reference Framework
The expectations on international interoperability

**Comment by christianheimann** [comment]
**3) Design Principles & Criteria**

**Comment by christianheimann** [comment]
**~~4) Lessons learned~~**
Hands-on Experience eID-Team;
Prototypes Hyperledger Indy Stack
OpenID and additional Elements
Analysis on BBS+, KERI/ACDC 
Dataflows (Transparence on where when what Data exists)
Open Questions

**Reply by christianheimann** [reply]
We put this chapter into annex.

**Comment by christianheimann** [comment]
**4) Scenarios**
Different Scenarios with the view for each scenario on:
- international context
- selected technologies
- characteristics
- feasability
- risks

**Comment by christianheimann** [comment]
**5) Public discussion of this paper and further process**
Description of the process with the leading question(s) to answer in the informal consultation.

**Comment by christianheimann** [comment]
**Please write general comments to the whole document here below!** in the sub-thread.

Have also a look at the complete text at: https://github.com/e-id-admin/open-source-community/blob/main/discussion-paper-tech-proposal/discussion-paper-tech-proposal.md
